sys/geom/geom_subr.c

```diff
@@ -1574,18 +1574,22 @@ DB_SHOW_COMMAND(geom, db_show_geom)
 				break;
 		}
 	} else {
-		switch (g_valid_obj((void *)addr)) {
+		switch (g_valid_obj((void *)(uintptr_t)addr)) {
```

Comment:
Eh, this probably doesn't work as-is.

Reply:
Oh, it sort of does, but g_valid_obj itself needs some changes to only compare addresses.